test/shared/src/main/scala/zio/test/PrettyPrint.scala

```diff
@@ -1,83 +1,77 @@
 package zio.test
 
-import zio.internal.ansi.AnsiStringOps
-import zio.{Chunk, NonEmptyChunk}
 import zio.stacktracer.TracingImplicits.disableAutoTrace
+import zio.{Chunk, NonEmptyChunk}
+
+import scala.annotation.switch
 
 /**
  * PrettyPrint will attempt to render a Scala value as the syntax used to create
  * that value. This makes it easier to copy-paste from values printed to the
  * console during tests back into runnable code.
  */
 private[zio] object PrettyPrint extends PrettyPrintVersionSpecific {
-  private val maxListLength = 10
   def apply(any: Any): String =
-    any match {
-      case nonEmptyChunk: NonEmptyChunk[_] =>
-        prettyPrintIterator(nonEmptyChunk.iterator, nonEmptyChunk.size, "NonEmptyChunk")
-      case chunk: Chunk[_] =>
-        prettyPrintIterator(chunk.iterator, chunk.size, "Chunk")
-      case array: Array[_] =>
-        prettyPrintIterator(array.iterator, array.length, "Array")
+    (any: @switch) match {
+      case null => "<null>"
+
+      case string: String =>
+        val surround = if (string.contains('\n')) "\"\"\"" else "\""
+        string.replace("\"", """\"""").mkString(surround, "", surround)
+
+      case int: Int               => int.toString
+      case long: Long             => long.toString
+      case double: Double         => double.toString
+      case float: Float           => float.toString
+      case boolean: Boolean       => boolean.toString
+      case char: Char             => s"'${char.toString}'"
+      case short: Short           => short.toString
+      case byte: Byte             => byte.toString
+      case bigDecimal: BigDecimal => bigDecimal.toString
+      case bigInt: BigInt         => bigInt.toString
+      case symbol: Symbol         => symbol.toString
 
       case Some(a) => s"Some(${PrettyPrint(a)})"
       case None    => s"None"
       case Nil     => "Nil"
```

Comment:
```suggestion
      case _: Nil     => "Nil"
```

The current code will actually match on `Seq.empty` for non-list based Seqs:
```
(Vector.empty[String]: Seq[String]) match {
  case Nil => "NIL"
  case _ => "EMPTY"
}
```

Reply:
Minor correction: Should be `_: Nil.type`

Reply:
Done. Thanks!